Library/Homebrew/brew.sh

```diff
@@ -296,6 +296,13 @@ source "$HOMEBREW_LIBRARY/Homebrew/utils/analytics.sh"
 setup-analytics
 report-analytics-screenview-command
 
+# Let user know we're still updating Homebrew if brew update --preinstall
+# exceeds 3 seconds.
+update-preinstall-timer() {
+  sleep 3
+  echo 'Updating Homebrew...' >&2
+}
```

Comment:
Yeh, that's fair enough.



> What kind of testing are you thinking about?

Just manually reproducing this locally. Amtrak made this easier today, though 😁